### DISCUSSION TITLE
How to edit attributes of assets produced by `load_assets_from_airbyte_instance`

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17484] [Q&A] [answered]
Hello i'm using `load_assets_from_airbyte_instance`, once this is run, is there any way to edit some attributes of the assets ? like adding Meta , or adding Upstream dependancies? 

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698669206340149?thread_ts=1698669206.340149&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
Here's an example of adding downstream dependencies: https://docs.dagster.io/integrations/airbyte#step-3-adding-downstream-assets similarly you can do so for adding upstream dependencies

**Reply by OmarSultan85** [reply]
Thanks a lot Yuhan, quick quetstion if you don't mind. load_assets_from_airbyte_instance usuing this, how do I itterate through the assets returned by this function? I want to create an asset_job for each of the assets returned by it
